R/github.R

```diff
@@ -11,18 +11,25 @@ github_response <- function(req) {
   parsed <- jsonlite::fromJSON(text, simplifyVector = FALSE)
 
   if (httr::status_code(req) >= 400) {
-    errors <- vapply(parsed$errors, `[[`, "message", FUN.VALUE = character(1))
-
-    stop(
-      parsed$message, " (", httr::status_code(req), ")\n",
-      paste("* ", errors, collapse = "\n"),
-      call. = FALSE
-    )
+    stop(oithub_error(req))
```

Comment:
Ugg, fixed in https://github.com/hadley/devtools/pull/1085/commits/d9a7d67c525b0495689832e22e597766d6e11481